src/_P039_Thermosensors.ino

```diff
@@ -234,6 +242,9 @@ boolean Plugin_039(byte function, struct EventStruct *event, String& string)
 
     case PLUGIN_INIT:
     {
+      initPluginTaskData(event->TaskIndex, new (std::nothrow) P039_data_struct(0u, 0u, 0u, 0u, 0u, false));
```

Comment:
Wouldn't it make more sense to add an extra constructor for this case, if you don't set any variable?

Reply:
Valid remark - solved with next push.